### PR TITLE
fix translations of metaedit dialog

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Follow up for #5646

### CODE REVIEW

**Comment by Marr11317** [comment]
wow sorry for that. I had no idea that `QT_TR_NOOP` only marked the strings for translation, without actually translating them.
Perhaps `openfilelocation::platformText()` could get back to metaedit.cpp

**Comment by Jojo-Schmitz** [comment]
> Perhaps `openfilelocation::platformText()` could get back to metaedit.cpp
What do you mean bythis?

> But your solution is cleaner if it works.
It only works half way it seems.

File: mscore/metaedit.cpp [hunk 1]
```diff
@@ -182,15 +182,15 @@ void MetaEditDialog::setDirty(const bool dirty)
...
 ///    Opens the file location with a QMessageBox::warning on failure
 //---------------------------------------------------------
 
 void MetaEditDialog::openFileLocation()
       {
       if (!OpenFileLocation::openFileLocation(filePath->text()))
-          QMessageBox::warning(this, QString(QT_TR_NOOP("Open Containing Folder Error")),
-                                     QString(QT_TR_NOOP("Could not open containing folder")));
+            QMessageBox::warning(this, QString(tr("Open Containing Folder Error")),
+                                       QString(tr("Could not open containing folder")));
```
Comment (dmitrio95) [inline mscore/metaedit.cpp:193]: These `QString()` conversions are not needed here anymore, `tr()` function already returns a `QString`.

Comment (Jojo-Schmitz) [reply mscore/metaedit.cpp:193]: True, fixed

**Comment by dmitrio95** [comment]
> It only works half way it seems.

The reason it doesn't show a translated tooltip text is probably that existing translations use a different context: the previously used [`QT_TR_NOOP`](https://doc.qt.io/qt-5/qtglobal.html#QT_TR_NOOP) macro assigns a context according to a class name (which is confirmed by Transifex where these strings are shown in `Ms::OpenFileLocation` context) while `QObject::tr()` doesn't use it. The solution with `QObject::tr()` should work as it is but it will require changing translation context for translated strings (which Transifex will probably do automatically).

Still if you want to keep the translated strings' context unchanged then something like `qApp->translate("Ms::OpenFileLocation", "Open Containing Folder")` should work. Another alternative is to use [`Q_DECLARE_TR_FUNCTIONS`](https://doc.qt.io/qt-5/qcoreapplication.html#Q_DECLARE_TR_FUNCTIONS) macro in `OpenFileLocation` class and replace `QObject::tr()` with plain `tr()` calls.

**Comment by Jojo-Schmitz** [comment]
Hmm, I tried with `Q_DECLARE_TR_FUNCTIONS()` and `qApp->translate()`, but in either case with out that "Ms::" before "OpenFileLocation". The latter BTW failed to even compile. Trying the former again...

**Comment by Jojo-Schmitz** [comment]
Yay! That's it, thanks!

**Pull request merged**